app/controller/requests.rb

```diff
@@ -0,0 +1,14 @@
+class Bnb < Sinatra::Base
+  get '/requests' do
+    @bookings = Booking.all(host_id: session[:user_id])
```

Comment:
This pulls all bookings requests associated with the current user. I have changed the variable to ```@host_requests``` for clarity.

Currently we display booking requests that are both confirmed and pending, so we haven't set ```@host_requests``` to sort out ```'Confirmed'``` or ```'Pending'``` requests. We could do this, but I don't really see a need at this point. Might be worth it later on when we refactor, though.